.github/workflows/label.yml

```diff
@@ -0,0 +1,12 @@
+name: Labeler
+on: [pull_request]
+
+jobs:
+  label:
+
+    runs-on: ubuntu-latest
```

Comment:
Why the empty lines around this one?

Reply:
https://github.com/actions/starter-workflows/blob/master/automation/label.yml

Reply:
Ho ok, fair enough.

I simply looked at the the README from your link (https://github.com/actions/labeler) where there were no spaces around this line.